resources/aws/peering.go

```diff
@@ -21,6 +24,8 @@ type VPCPeeringConnection struct {
 	PeerVPCId string // PeerVPCId the ID of the VPC in the host cluster.
 	id        string
 	AWSEntity
+
+	Logger micrologger.Logger
```

Comment:
I also added a logger, so we can get some visibility on the VPC acceptance retries.